CHANGELOG.asciidoc

```diff
@@ -8,6 +8,7 @@
 // Template, add newest changes here
 
 === Beats version HEAD
+https://github.com/elastic/beats/compare/v5.0.0-rc1...master[Check the HEAD diff]
```

Comment:
It looks like you made no changes to this file, so can you reset this file so that there are no changes in the diff.